### PR TITLE
Changed VersionsCommand to give more info

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

I changed the /sf version command to show a little more info
https://i.imgur.com/hENZ5nk.png

## Changes

Changed ReflectionUtils.getVersion() to
Bukkit.getVersion()

## Related Issues

N/A
## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by WalshyDev — COMMENTED** [review]
This will be helpful and it still shows the MC version.

What does Spigot look like though? Do we need special handling for them?

**Comment by J3fftw1** [comment]
> 
> 
> This will be helpful and it still shows the MC version.
> 
> What does Spigot look like though? Do we need special handling for them?

https://i.imgur.com/OhcxNrm.png

Spigot doesnt look that good but it does show the full version though

**Review by WalshyDev — APPROVED** [review]
LGTM - Paper and Spigot both show MC version still along with relevant build info. I'm happy

**Comment by J3fftw1** [comment]
Also changed the unofficial naming. Walshy complained about it in my dm's.

**Review by WalshyDev — APPROVED** [review]
LGTM - Paper and Spigot both show MC version still along with relevant build info. I'm happy

File: pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
     <!-- Our default version will be UNOFFICIAL, this will prevent auto updates -->
     <!-- from overriding our local test file -->
-    <version>4.8-UNOFFICIAL</version>
+    <version>UNOFFICIAL</version>
```
Comment (TheBusyBiscuit) [inline pom.xml:12]: no... just why...

File: src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/VersionsCommand.java [hunk 2]
```diff
@@ -22,14 +22,14 @@
     }
 
     @Override
-    public void onExecute(CommandSender sender, String[] args) {
+    public void onExecute(@Nonnull CommandSender sender, @Nonnull String[] args) {
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/VersionsCommand.java:25]: :+1:

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Not too sure about the aesthetics.
Usually the Server software version isn't of much concern but it recently has become one, so I like the addition of that.
One could of course extract the specific build and only show that... but I think that'd probably be too much effort for such a small change.

Could you perhaps test it on one or two more softwares though, Tuinity perhaps? As that's the most popular one after Spigot and Paper.
I just wanna be sure, we won't be sawing off the branch we stand on here.
Otherwise i like the addition, will be very useful!

File: pom.xml (hunk 1 above)
Comment (J3fftw1) [reply pom.xml:12]: Also changed the unofficial naming. Walshy complained about it in my dm's.

**Comment by J3fftw1** [comment]
> 
> 
> Not too sure about the aesthetics.
> Usually the Server software version isn't of much concern but it recently has become one, so I like the addition of that.
> One could of course extract the specific build and only show that... but I think that'd probably be too much effort for such a small change.
> 
> Could you perhaps test it on one or two more softwares though, Tuinity perhaps? As that's the most popular one after Spigot and Paper.
> I just wanna be sure, we won't be sawing off the branch we stand on here.
> Otherwise i like the addition, will be very useful!

https://cdn.discordapp.com/attachments/627865620947402793/797556227944480798/unknown.png
i tested yatopia aswell if thats okay?

File: pom.xml (hunk 1 above)
Comment (WalshyDev) [reply pom.xml:12]: No reason for this to still have 4.8- at the start NotLikeThis

Comment (TheBusyBiscuit) [reply pom.xml:12]: Revert this change please.
Having an actual semantic version has lots of reasons. Mainly to differentiate between versions (duh...).
But also as the "actual" version in front of the `-UNOFFICIAL` has often worked to discourage people from changing this field.

Comment (J3fftw1) [reply pom.xml:12]: i will change it back iguess

**Comment by J3fftw1** [comment]
https://i.imgur.com/GZUbAHY.png
couldnt be bothered to start mc again

**Review by TheBusyBiscuit — APPROVED** [review]
An additional test for this on Tuinity would be perfect but it looks good as is.

**Comment by TheBusyBiscuit** [comment]
> https://i.imgur.com/GZUbAHY.png
> couldnt be bothered to start mc again

Perfect 👌🏻

**Comment by J3fftw1** [comment]
> 
> 
> An additional test for this on Tuinity would be perfect but it looks good as is.

i did XD check message above yours

**Comment by TheBusyBiscuit** [comment]
> > An additional test for this on Tuinity would be perfect but it looks good as is.
> 
> i did XD check message above yours

And I replied XD check message above yours :P

**Comment by J3fftw1** [comment]
> 
> 
> > > An additional test for this on Tuinity would be perfect but it looks good as is.
> > 
> > 
> > i did XD check message above yours
> 
> And I replied XD check message above yours :P

i was about to reply again LMFAO

**Review by WalshyDev — APPROVED** [review]
LGTM

**Pull request merged**